styles/style.css

```diff
@@ -94,13 +95,294 @@ body {
+  background-color: #212121;
+  width: 100%;
+  height: 1px;
+}
+
+/* Work Cards */
+.work-card {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  width: 100%;
+  margin-top: 20%;
+}
+
+.card-title {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 1.5em;
+  font-weight: 150;
+  margin: 4% 0;
+}
+
+.card-text {
+  font-family: 'Inter', sans-serif;
+  margin-bottom: 3%;
+  font-size: 16px;
+  font-weight: 400;
+}
+
+/* Buttons */
+.btn-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+  margin-bottom: 2%;
+}
+
+li {
   list-style: none;
 }
+
+.feature-btn {
+  width: 49px;
+  height: 40px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.ruby {
+  margin-right: 20%;
+}
+
+.project-btn {
+  width: 35%;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: 1px solid #8993a4;
+  text-align: center;
+  font-family: 'inter', sans-serif;
+  font-weight: 700;
+  font-size: 1em;
+}
+
+.project-btn:hover {
+  background-color: #fff5e1;
+  color: #f55800;
+  transition: 0.3s ease-in-out;
+  box-shadow: 0 0 1em #ff6b00;
+}
+
+/* Project Cards */
+.card-container {
+  width: 100%;
+  margin: 10% 0;
+  grid-column: 2;
+  background-image: url(../media/project-img.png);
+  background-size: 100%;
+  height: 340px;
+  background-repeat: no-repeat;
+}
+
+.card-content {
+  margin-top: 30%;
+}
+
+.project-title {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 2em;
+  color: #fff;
+  margin-left: 4%;
+}
+
+.project-text {
+  margin-left: 4%;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  color: #fff;
+  font-size: 16px;
+}
+
+.project-btn-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+  margin-top: 2%;
+}
+
+.project-feature-btn {
+  width: 49px;
+  height: 40px;
+  background-color: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+  font-weight: 500;
+  font-size: 15px;
+  color: #fff;
+  margin-top: 4%;
+}
+
+.html-list {
+  margin-left: 4%;
+}
+
+.boostrap {
+  width: 95px;
+  height: 40px;
+}
+
+.project-ruby {
+  margin-right: 25%;
+}
+
+.project-final-btn {
+  width: 100%;
+  height: 48px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: 1px solid #8993a4;
+  text-align: center;
+  font-family: 'inter', sans-serif;
+  font-weight: 700;
+  font-size: 1em;
+  margin-top: 5%;
+}
+
+.project-final-btn:hover {
+  background-color: #fff5e1;
+  color: #f55800;
+  transition: 0.3s ease-in-out;
+  box-shadow: 0 0 1em #ff6b00;
+}
+
+.last-card {
+  margin-bottom: 70%;
+}
+
+/* About Me Section */
+.aboutme-section {
+  margin-top: 40%;
+  background-image: url(../media/aboutme-background.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: 17em;
+}
```

Comment:
- [ ] KIndly make sure the `About Me` section contents are aligned on the page using `flexbox` in order to meet the project  requirement  :wink: